### PR TITLE
Revert "removes dab"

### PR DESCRIPTION
**Pull request by maxion12345**
Reverts yogstation13/Yogstation#9733

hopek showed me where this button is and told me I need to "make a good reason"

seeing as our transition to mrp has fallen flat on its face and the RP rules document is probably accessed less then the space inbetween a nun's thighs, I believe its time to bring back dabbing as a way for our players to express how fucking stupid they are. (mainly me)

also, someone needs to add *cripwalk emote

(serious part)
The main reason this is being reverted is that its removal didn't do anything in the first place. It was touted as the beginning of the end of LRP and the first step on a path to usher in a great vision of MRP, but we all know how that went. I think it's time to face the reality of our low-to-medium-rp server and bring this back. 

Yogstation is, and always will be, this weird conglomeration of people that enjoy RP and people that enjoy doing things that aren't RP. Why can't we cater to both? If you want to do your RP, go ahead. If you want to dab every time you triumph over the local shitter clown, why not?

### CODE REVIEW

**Comment by Cartlord** [comment]
Is it still possible for them to *dab and get the "Voice of God" trauma? 'Cause it might be a good idea to change that!

**Comment by maxion12345** [comment]
> Is it still possible for them to *dab and get the "Voice of God" trauma? 'Cause it might be a good idea to change that!

nah they removed that

**Comment by Cartlord** [comment]
based

**Comment by N3D6** [comment]
yeah

**Comment by maxion12345** [comment]
inb4 council vote to revert

**Comment by JamieD1** [comment]
As there was a council vote to remove it, there will need to be one to reinstate it.

**Comment by JamieD1** [comment]
Awaiting Vote 824.

**Comment by Hopekz** [comment]
@wejengin2 where did the controversial tag come from?

There isn't a single thing posted here on why not to do it.

**Comment by patpol4** [comment]
> There isn't a single thing posted here on why not to do it.

Yes but it has proven to be a controversial topic in the past, so much so that it needed a council vote to remove a simple feature.

**Comment by boba247** [comment]
The brain damage should be left out. If someone wants to dab fine. If someone wants to dab in hopes for a funny brain damage trauma not so fine.

File: yogstation/code/modules/mob/living/emote.dm
Comment (adamsong) [inline yogstation/code/modules/mob/living/emote.dm]: Remove the brain damage, we don't want people spamming for traumas

**Review by adamsong — CHANGES_REQUESTED** [review]

**Review by adamsong — APPROVED** [review]
Code looks fine, approving but don't merge pending council vote

**Comment by JamieD1** [comment]
824 Approved. 
https://forums.yogstation.net/threads/vote-824-reverts-council-vote-https-forums-yogstation-net-threads-vote-722-remove-dab-22271-with-pr-https-github-com-yogstation13-yogstat.23702/

**Pull request merged**
